tests/Unit/FramesMonitor/constructor.test.js

```diff
@@ -177,7 +177,8 @@ describe('FramesMonitor::constructor', () => {
             bufferMaxLengthInBytes     : config.bufferMaxLengthInBytes,
             errorLevel                 : config.errorLevel,
             exitProcessGuardTimeoutInMs: config.exitProcessGuardTimeoutInMs,
-            timeout                    : config.timeoutInMs * 1000
+            timeout                    : config.timeoutInMs * 1000,
+            analyzeDuration            : config.analyzeDurationInMs && config.analyzeDurationInMs * 1000 || 0
```

Comment:
No logic in test data. Test data must be clear and without any variance and side effects. So, it's better to specify it explicitly in the `config`.

Reply:
was improved